tests/testthat/test-duplicate_argument_linter.R

```diff
@@ -1,82 +1,144 @@
+  expect_lint(
+    "fun(arg = 1)",
     NULL,
-    duplicate_argument_linter())
+    duplicate_argument_linter()
+  )
 
-  expect_lint("fun('arg' = 1)",
+  expect_lint(
+    "fun('arg' = 1)",
     NULL,
-    duplicate_argument_linter())
+    duplicate_argument_linter()
+  )
 
-  expect_lint("fun(`arg` = 1)",
+  expect_lint(
+    "fun(`arg` = 1)",
     NULL,
-    duplicate_argument_linter())
+    duplicate_argument_linter()
+  )
 
-  expect_lint("'fun'(arg = 1)",
+  expect_lint(
+    "'fun'(arg = 1)",
     NULL,
-    duplicate_argument_linter())
+    duplicate_argument_linter()
+  )
 
-  expect_lint("(function(x, y) x + y)(x = 1)",
+  expect_lint(
+    "(function(x, y) x + y)(x = 1)",
     NULL,
-    duplicate_argument_linter())
+    duplicate_argument_linter()
+  )
 
-  expect_lint("dt[i = 1]",
+  expect_lint(
+    "dt[i = 1]",
     NULL,
-    duplicate_argument_linter())
+    duplicate_argument_linter()
+  )
 
-  expect_lint("fun(arg = 1, arg = 2)",
+  expect_lint(
+    "fun(arg = 1, arg = 2)",
     list(message = rex("Duplicate arguments in function call.")),
-    duplicate_argument_linter())
+    duplicate_argument_linter()
+  )
 
-  expect_lint("fun(arg = 1, 'arg' = 2)",
+  expect_lint(
+    "fun(arg = 1, 'arg' = 2)",
     list(message = rex("Duplicate arguments in function call.")),
-    duplicate_argument_linter())
+    duplicate_argument_linter()
+  )
 
-  expect_lint("fun(arg = 1, `arg` = 2)",
+  expect_lint(
+    "fun(arg = 1, `arg` = 2)",
     list(message = rex("Duplicate arguments in function call.")),
-    duplicate_argument_linter())
+    duplicate_argument_linter()
+  )
 
-  expect_lint("'fun'(arg = 1, arg = 2)",
+  expect_lint(
+    "'fun'(arg = 1, arg = 2)",
     list(message = rex("Duplicate arguments in function call.")),
-    duplicate_argument_linter())
+    duplicate_argument_linter()
+  )
 
-  expect_lint("(function(x, y) x + y)(x = 1, x = 2)",
+  expect_lint(
+    "(function(x, y) x + y)(x = 1, x = 2)",
     list(message = rex("Duplicate arguments in function call.")),
-    duplicate_argument_linter())
+    duplicate_argument_linter()
+  )
 
-  expect_lint("dt[i = 1, i = 2]",
+  expect_lint(
+    "dt[i = 1, i = 2]",
     list(message = rex("Duplicate arguments in function call.")),
-    duplicate_argument_linter())
+    duplicate_argument_linter()
+  )
 
   expect_lint(
     "list(
       var = 1,
       var = 2
     )",
     list(message = rex("Duplicate arguments in function call.")),
-    duplicate_argument_linter())
+    duplicate_argument_linter()
+  )
 
   expect_lint(
     "list(
       var = 1,
       var = 2
     )",
     NULL,
-    duplicate_argument_linter(except = "list"))
+    duplicate_argument_linter(except = "list")
+  )
 
   expect_lint(
     "(function(x, y) x + y)(x = 1)
     list(var = 1, var = 2)",
     NULL,
-    duplicate_argument_linter(except = "list"))
+    duplicate_argument_linter(except = "list")
+  )
 
   expect_lint(
     "fun(`
-` = 1, `
-` = 2)",
+     ` = 1, `
```

Comment:
is this a styler bug? the nature of this test has changed -- on `main`, it's testing that the variable named `\n` gets caught when used as a duplicate. Now the test still checks whitespace-only names, but adds spaces...

https://github.com/r-lib/lintr/blob/e993d4d37fc3016aefade264b4547ea76dc40f06/tests/testthat/test-duplicate_argument_linter.R#L73-L75

Reply:
No, not styler bug. I unintentionally changed this. Reverted.